server/balancer_test.go

```diff
@@ -216,50 +277,43 @@ func (s *testBalanceStorageSchedulerSuite) TestBalance(c *C) {
 	c.Assert(sb.Schedule(cluster), IsNil)
 	opt.SetMaxReplicas(1)
 	c.Assert(sb.Schedule(cluster), NotNil)
-
-	// Test MinBalanceDiffRatio.
-	// When diff storage ratio < MinBalanceDiffRatio, no schedule.
-	tc.updateRegionCount(2, 6, 0.4)
-	tc.updateRegionCount(3, 7, 0.4)
-	tc.updateRegionCount(4, 8, 0.4)
-	c.Assert(sb.Schedule(cluster), IsNil)
 }
 
-func (s *testBalanceStorageSchedulerSuite) TestReplicas3(c *C) {
+func (s *testBalanceRegionSchedulerSuite) TestReplicas3(c *C) {
```

Comment:
Some comments should be updated in this test case. 'stroage ratio/region ratio' -> 'region count' or 'region score'.